projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -189,6 +204,8 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
   @Input() summaryRow: boolean;
   @Input() summaryPosition: string;
   @Input() summaryHeight: number;
+  @Input() rowDraggable: boolean;
```

Comment:
Does it make sense to have this on a global level? Shouldn't this be on every individual row?

Reply:
changed to be at row level